Assets/Scripts/Screeps3D/Player/PlayerTransporter.cs

```diff
@@ -27,6 +27,8 @@ private void OnChoose(Room room)
 
             if (_playerGaze != null)
             {
+                _playerGaze.allowFocus = false;
+                CameraRig.Instance.OnTargetReached += OnTargetReached;
                 // disable player gaze while traveling to new room looks like we are subscribing to all the rooms we are scrolling past.
                 // while this prevents the subscription of too many rooms, it also breaks the subscriptions in general so they are not behaving as expected   
                 //CameraRig.Instance.OnTargetReached += OnTargetReached;
```

Comment:
Might want to make sure we do not leave stale comments in the codebase. are theese comments still relevant with your fix?